lib/services/keyVault/lib/models/jsonWebKey.js

```diff
@@ -149,7 +150,7 @@ JsonWebKey.prototype.mapper = function () {
         },
         k: {
           required: false,
-          serializedName: 'K',
+          serializedName: 'k',
```

Comment:
Can this cause serialization breaking change for folks who have serialized the key?

---
In reply to: [76497661](https://github.com/Azure/azure-sdk-for-node/pull/1858#discussion_r76497661) [](ancestors = 76497661,76497637)